src/uct/ib/mlx5/ib_mlx5.inl

```diff
@@ -13,6 +13,13 @@ uct_ib_mlx5_get_cqe(uct_ib_mlx5_cq_t *cq,  unsigned index)
     return cq->cq_buf + ((index & (cq->cq_length - 1)) << cq->cqe_size_log);
 }
 
+static UCS_F_ALWAYS_INLINE int
+uct_ib_mlx5_no_cqe(uint8_t op_own, unsigned index, unsigned mask)
+{
+    return ((op_own & MLX5_CQE_OWNER_MASK) == !(index & mask)) ||
```

Comment:
may be
```
!!(op_own & MLX5_CQE_OWNER_MASK) == !(index & mask)
```

Reply:
imo, not needed, because owner bit is always the most right according to CQE format definition (so MLX5_CQE_OWNER_MASK is just `1`)